Libraries/Opc.Ua.Security.Certificates/X509Crl/X509Crl.cs

```diff
@@ -250,7 +250,14 @@ internal void DecodeCrl(byte[] tbs)
                     // Signature Algorithm Identifier
                     AsnReader sigReader = seqReader.ReadSequence();
                     string oid = sigReader.ReadObjectIdentifier();
-                    m_hashAlgorithmName = Oids.GetHashAlgorithmName(oid);
+                    try
+                    {
+                        m_hashAlgorithmName = Oids.GetHashAlgorithmName(oid);
+                    }
```

Comment:
do not hide this exception, we can't handle it, the result is unpredictable otherwise

Reply:
but is the hash algorithm name even important for a crl?
Our Oids.GetHashAlgorithmName function does only support a few selected hash algorithms, maybe we should extend it with the most common ones if it is used for crls?

Reply:
better is to not accept these CRL, hashes of these algorithms are considered unsecure.